packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -42,6 +47,17 @@ export const buildIframeSrc = (params: BuildIframeSrcOptions): string => {
     .forEach(([paramName, paramValue]) => {
       url.searchParams.set(paramName, paramValue)
     })
+  if (embedId) {
+    url.searchParams.set('typeform-embed-id', embedId)
+  }
+
+  const queryWithTransitiveSearchParams = getTransitiveSearchParams(queryParams, options.transferableUrlParameters)
```

Comment:
Do we need this? I think `mapOptionsToQueryParams` already takes care of adding transitive search params.